sdk_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/joho/godotenv"
+)
+
+func TestInitializeClient(t *testing.T) {
+	_ = godotenv.Load(".env")
+	_, err := pubsub.NewClient(context.Background(), ProjectID)
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Again, why are you testing the pubsub library instead of the functions you wrote?
I genuinely don't understand what's going on. Please explain

Reply:
I'll redo the testing

No explanation can justify what I did.

I'm sorry for the below average work

Reply:
Man, it’s not that deep. Hahaha. 
just call the functions in the test, instead of copying the implementation. 

Also, use an assertion library to assert values in your tests: https://github.com/stretchr/testify